core-bundle/src/Migration/Version413/RelLightboxMigration.php

```diff
@@ -86,6 +80,25 @@ public function run(): MigrationResult
 
     private function getTargets(): array
     {
-        return array_map(static fn (string $target) => explode('.', $target), self::$targets);
+        $this->framework->initialize();
+        $schemaManager = $this->connection->createSchemaManager();
+
+        $targets = [];
+
+        foreach ($schemaManager->listTableNames() as $tableName) {
+            try {
+                Controller::loadDataContainer($tableName);
+            } catch (\Throwable $e) {
+                continue;
+            }
+
+            foreach ($GLOBALS['TL_DCA'][$tableName]['fields'] ?? [] as $fieldName => $fieldConfig) {
+                if ('tiny' === substr($fieldConfig['eval']['rte'] ?? '', 0, 4)) {
```

Comment:
should probably use `str_starts_with`?

Reply:
Yes, we could do that. But let‘s try not to use these features until we can use them natively. The current solution is working and Rector will automatically change it to `str_starts_with` when merging upstream.